tools/operator/connectionprofile/updateconncectionprofile.go

```diff
@@ -63,15 +63,43 @@ func (c ConnProfile) updateConnectionProfilePerChannel(inputObject interface{},
 func (c ConnProfile) updateConnectionProfilePerOrg(organizations []inputStructs.Organization, inputArgs ...string) error {
 
 	var err error
+	connectionProfilesList := []string{}
 	action, orgName, channelName := inputArgs[0], inputArgs[1], inputArgs[2]
```

Comment:
need to sanity check the length of inputArgs

Reply:
I think it's not necessary to check the length. Because we are passing exactly three elements in that array in the function where we call it.

Reply:
It is necessary to check the length here in order to make the code easy to maintain.

Reply:
okay. Will change this